src/rtk/features/singleRights/singleRightsSlice.ts

```diff
@@ -185,47 +194,92 @@ const singleRightSlice = createSlice({
   },
   extraReducers: (builder) => {
     builder
+      .addCase(delegationAccessCheck.pending, (state, action) => {
```

Comment:
This code is hard to read but I understand that it has to be that way. Some json doc comments can help making it clearer to understand it.

Reply:
Understandable
Does this work as an explanation?
![image](https://github.com/Altinn/altinn-access-management-frontend/assets/22012997/7f94b5ba-f891-432d-956f-555cadda3845)

Reply:
Ja det der er bra!